src/test/resources/test_data/clear-all-data.sql

```diff
@@ -4,7 +4,6 @@ DELETE from additional_condition_upload_detail;
 DELETE from additional_condition_upload_summary;
 DELETE from additional_condition;
 DELETE from bespoke_condition;
-DELETE from licence_history;
```

Comment:
Want to delete the licence events instead?

Reply:
No, its ok, there's a cascading delete for events when deleting the licence.

Reply:
Audit events do not cascade deletes - we keep this regardless, but licence events are always linked with a licence and do.